### ISSUE TITLE
Questions about the setting

### ISSUE THREAD

**Issue by viridyu** [yaq007/Autofocus-Layer#5]
Thank you for your sharing.

I have a question about the model setting. I want to train another new model for BRATS2015 dataset. Are the default setting in the code as same as that reported in your MICCAI2018 paper? For example, when I use your default setting, I find the learning rate decays since the first epoch and the batch size is 10.

Look forward to your reply. Thank you.

**Comment by yaq007** [comment]
I think the default setting is the setting we used.
